configuration/concepts/misc.csv

```diff
@@ -22,4 +22,6 @@ f36097e7-7d66-4196-8d95-6b66f31435cc,,,See comments,See comments,Commentaires,Co
 26c0366b-83e4-4401-9699-e545219b8870,,CIEL:5622,Other,Other,Autre,Autre,,,Misc,N/A,,,,,,,,,,,
 75e366a1-c441-4c94-9c53-312c9b8cc550,,CIEL:1175,Not Applicable,Not Applicable,Non applicable,Non applicable,,,Misc,N/A,,,,,,,,,,,
```

Comment:
Short name should be "N/A" in en and fr.

Reply:
Done